app/views/ShareListView/Header.js

```diff
@@ -84,6 +85,12 @@ class ShareListHeader extends PureComponent {
 					/>
 				</View>
 			);
+		} else if (isIOS) {
```

Comment:
Why do we have a `SearchHeader` component for iOS and a TextInput on Android?
It should be the component, if necessary.